friends/src/Components/NewFriendForm.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import ReactDOM from 'react-dom'
+import axios from 'axios'
+
+export default class NewFriendForm extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            input_firstName: '',
+            input_lastName: '',
+            input_age: '',
+            input_email: ''
+        }
+    }
+    // CHANGE HANDLERS 
+        changeHandler_firstName = e => {
```

Comment:
DRYer implementation - a more dynamic `changeHandler`
```suggestion
        changeHandler = e => this.setState({[e.target.name]: e.target.value})
```